cmd/csaf_checker/processor.go

```diff
@@ -1082,27 +1075,37 @@ func (p *processor) checkProviderMetadata(domain string) error {
 	return nil
 }
 
-// checkSecurity checks the security.txt file by making HTTP request to fetch it.
+// check810 checks the security.txt file by making HTTP request to fetch it.
 // It checks the existence of the CSAF field in the file content and tries to fetch
 // the value of this field. As a result of these a respective error messages are
 // passed to the badSecurity method in case of errors.
-// It returns nil if all checks are passed.
-func (p *processor) checkSecurity(domain string) error {
+// Then it checks if the "csaf.data.security.domain.tld" DNS record is available
```

Comment:
The order should be as in https://docs.oasis-open.org/csaf/csaf/v2.0/cs02/csaf-v2.0-cs02.html#731-finding-provider-metadatajson :
 1. check well-known
 2. then security.txt
 3. DNS  (only be considered fatal, if 1. and 2. did not lead to a location of a PMD.)  We do it anyway to check if we find the same PMD there and inform about it.